### ISSUE TITLE
Improvement: suggest users to change Fantom RPC

### ISSUE THREAD

**Issue by fameal** [fameal/ape-tax#112]
When the RPC that the user has for Fantom is not compatible with our APY calculations, suggest to change it to `rpc.ftm.tools`.

**Comment by TBouder** [comment]
I don't think we can do it in an automatic way, this would require a popup or something like this

**Comment by fameal** [comment]
We can leave for when we have docs
